CMakeLists.txt

```
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(BigBaseV2 CXX ASM_MASM)
+
+set(SRC_DIR "${PROJECT_SOURCE_DIR}/BigBaseV2/src")
+
+set(CMAKE_CONFIGURATION_TYPES Release)
+
+
+# Fetch content
+message("\nFetching modules")
+include(scripts/MinHook.cmake)
+include(scripts/g3log.cmake)
+include(scripts/pugixml.cmake)
+include(scripts/json.cmake)
+
+message("\nFetching custom modules")
+include(scripts/ImGui.cmake)
```

```suggestion
include(scripts/imgui.cmake)
```

case-sensitive (at least on unix)